nms/app/packages/magmalte/grafana/dashboards/CWFDashboards.js

```diff
@@ -100,8 +100,45 @@ export const CWFSubscriberDBData: GrafanaDBData = {
           description:
             "Should just be 0 for subscribers without an active session and 1 for those with. Not sure what's up with this now.",
         },
+        {
+          title: 'User Growth',
+          targets: [
+            {
+              expr: 'count(sum(session_start{msisdn=~".*"}) by (msisdn))',
+              legendFormat:
+                '',
+            },
+          ],
+          description:
+            "Number of unique subscribers over time",
+        },
+        {
+          title: 'Locally Terminated Sessions',
+          targets: [
+            {
+              expr: 'count(sum(session_manager_terminate{msisdn=~"$msisdn"}) by (msisdn))',
+              legendFormat:
+                '',
+            },
+          ],
+          description:
+            "Number of subscriber sessions terminated locally",
+        },
+        {
+          title: 'Users who used data',
+          targets: [
+            {
+              expr: 'count(sum({msisdn=~"$msisdn", direction="down"}) by (msisdn))',
```

Comment:
Also the actual metric is missing here.

Reply:
My bad. I think it got missed while i was doing some lint efforts. Nice catch. Thanks.

Reply:
You should use some threshold value to avoid counting people with a value of 0 (idk if that metric would be created in the first place, but it's good practice anyways.

`count(sum(<metric>{labels} > <threshold_bytes>) by (msisdn)`

Reply:
Ok. Will make the relevant change.